packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -83,6 +83,8 @@ Yes. When you turn the custom domain on, the Okta domain (for example, `example.
 
 This method of configuring a custom domain is recommended because Okta manages your certificate renewals in perpetuity through an integration with Let's Encrypt, which is a free certificate authority. The certificate procurement process is free, and also faster and easier than configuring a custom domain with your own TLS certificate.
 
+> **Note:** If your custom domain is currently using your own TLS certificate and you'd like to migrate it to use an Okta-managed certificate, contact support@okta.com to request a seamless migration with no downtime.
```

Comment:
```suggestion
> **Note:** If your custom domain uses your own TLS certificate and you want to migrate to an Okta-managed certificate, contact your Okta account team or post on our [forum](https://devforum.okta.com/) to request a seamless migration with no downtime.
```

Reply:
We've been asked not to include the support email in the docs. From our style guide:

> To provide a CTA for those looking for support, use contact your Okta account team or ask us on our [forum](https://devforum.okta.com/). Do not use the Okta support email.